WcaOnRails/config/routes.rb

```diff
@@ -294,8 +294,11 @@
       get '/search/users' => 'api#users_search'
       get '/search/regulations' => 'api#regulations_search'
       get '/search/incidents' => 'api#incidents_search'
-      get '/users/:id' => 'api#show_user_by_id', constraints: { id: /\d+/ }
-      get '/users/:wca_id' => 'api#show_user_by_wca_id'
+      get '/users' => 'users#show_users_by_id'
+      get '/users/me' => 'users#show_me'
```

Comment:
How is this route different from `api/v0/me`?

Reply:
api/v0/me is just available for oauth users